backup_atat_provisioning_wizard_api.yaml

```diff
@@ -0,0 +1,737 @@
+openapi: 3.0.2
```

Comment:
I assume we don't want this file right?

Reply:
We need a backup because there are still functions that don't have `x-amazon-api-gateway-integration` (because they aren't defined in the atat-web-api. For example, there is AT-6494 (my TODO task), which is to create the POST for submitting the portfolio: this is defined in the yaml spec, but not in the atat-web-api, so it doesn't have a lambda function to reference. We can't compile it because it doesn't have a x-amazon-api-gateway-integration, so I removed it from the yaml, and made a backup for when we implement it. This probably isn't the best way to do this.

Reply:
What if we just don't provide `x-amazon-api-gateway-*` attributes until there's an implementation ready?

Reply:
In that case, it won't work with `cdk deploy`. It throws a `missing amazon integration for ......` the endpoint.

Reply:
We could have 2 yaml files, one with the integrations, and the other just openapi 3.0 yaml. We coan point the web stack at the one with integrations.